packages/sanes-chrome-extension/src/logic/persona/accountManager.ts

```diff
@@ -1,10 +1,10 @@
-import { Algorithm, ChainId, publicIdentityEquals, PublicIdentity } from '@iov/bcp';
+import { Algorithm, ChainId, PublicIdentity, publicIdentityEquals } from '@iov/bcp';
 import { UserProfile, WalletId } from '@iov/core';
 import { Slip10RawIndex } from '@iov/crypto';
 import { ReadonlyWallet } from '@iov/keycontrol/types/wallet';
 
 export interface AccountInfo {
```

Comment:
As far as I see this interface is only used here, so you can remove the `export`, right?